nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -51,6 +53,9 @@ def setup_experiment(self, config):
                                   times, i.e. the mode, will be chosen for the
                                   meta-testing phase.
             - num_meta_testing_runs: number of meta-testing phases to run
+            - meta_test_sample_size: number of images per class to sample from for
```

Comment:
The naming doesn't make clear it is for training - shouldn't be meta_test_train_sample_size? I know it is a ridiculously long name, but we can drop the meta instead, just `test_train_sample_size`.

Reply:
Good point. I'll update this name as well as `meta_train_sample_size` -> `train_train_sample_size`.